scripts/js/commands/checkStaleImages.ts

```diff
@@ -70,8 +74,8 @@ async function getStrippedImagePaths(): Promise<Set<string>> {
 
 async function isImageUnused(strippedFp: string): Promise<boolean> {
   const args = (await isInstalled("rg"))
-    ? ["rg", "-l", strippedFp, "{docs,learning}", "-g", "!docs/api"]
-    : ["git", "grep", "-l", strippedFp, "{docs,learning}", ":!docs/api"];
+    ? ["rg", "-l", strippedFp, "docs", "learning", "-g", "!docs/api"]
+    : ["git", "grep", "-l", strippedFp, "docs", "learning", ":!docs/api"];
```

Comment:
I tested this script with both tools by replacing `(await isInstalled("rg"))` with a bool.

<img width="696" alt="Screenshot 2025-05-06 at 15 03 00" src="https://github.com/user-attachments/assets/76cb2cf0-92ac-4321-b551-ed03382252ba" />